pkg/client/clientset/clientset.go

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Can you document the command you used to generate this?
Maybe add a script `hack/update-generated-clientset.sh`?